tests/github-discussion-schema.test.mjs

```diff
@@ -26,11 +26,13 @@ const filesInDir = fs
 
 filesInDir.forEach((file) => {
   const content = fs.readFileSync(join(GH_DATA, file)).toString();
-  const data = JSON.parse(stripJsonComments(content));
+  if (content !== "null") {
```

Comment:
Why check for contents of the file being "null"? When would this ever happen?

Reply:
When  storing a logic of github discussions, if there is no discussions found then empty fill is created so null indicates empty file and if we do not create empty file then this way also testing will fail .

Reply:
skip creating the file altogether if nothing is there to be written instead

Reply:
Then also the test will be failed, won't it?

Reply:
`filesInDir` would result in empty array right? So why would it fail?

Reply:
So for that we have to create an empty directory?

Reply:
You can simply skip the whole write step altogether when there is no data. Empty directories won't get committed to git anyhow.

Reply:
Okk skip the whole write step but if there is no discussions directory then also test will be fail
![image](https://github.com/user-attachments/assets/980c904f-4b52-48dd-b395-bc6de0ac9031)

Reply:
Run the test only if the directory is present then?